google/cloud/storage/README.md

```diff
@@ -1,22 +1,46 @@
 # Google Cloud Storage
 
-[![Documentation][doxygen-shield]][doxygen-link]
-
-[doxygen-shield]: https://img.shields.io/badge/documentation-master-brightgreen.svg
-[doxygen-link]: http://googleapis.github.io/google-cloud-cpp/latest/storage
+Cloud Storage allows world-wide storage and retrieval of any amount of data at
+any time. You can use Cloud Storage for a range of scenarios including serving
+website content, storing data for archival and disaster recovery, or
+distributing large data objects to users via direct download.
 
 This directory contains the implementation of the Google Cloud Storage C++
 client.
 
+[![Documentation][doxygen-shield]][doxygen-link]
+
+[doxygen-shield]: https://img.shields.io/badge/documentation-master-brightgreen.svg
+[doxygen-link]: https://googleapis.github.io/google-cloud-cpp/latest/storage
+[quickstart-link]: https://googleapis.github.io/google-cloud-cpp/latest/storage/index.html
+[gcs-docs-link]: https://cloud.google.com/storage/docs
+
 ## Status
 
 This library support Google Cloud Storage at the
-[Beta](../README.md#versioning) quality level.
+[Beta](../README.md#versioning) quality level. Please note that, as is often the
+case with C++ libraries, we do **not** follow semantic versioning in the Cloud
+C++ client libraries. We make every effort to document backwards-incompatible
+API changes in the [release notes](#release-notes) below. 
 
 ## Documentation
 
-The [reference guide][doxygen-link], can be found online. The top-level page in
-the reference guide includes a quick start guide.
+* The [client library reference][doxygen-link] contains full documentation
+  for the Cloud Storage C++ client library, and includes code samples for all
+  APIs.
+* A [quick start][quickstart-link] is part of the client library reference.
+* Please consult the [Cloud Storage website][gcs-docs-link] for general
+  documentation on Cloud Storage features, APIs, other client libraries, etc.
+
+## Contributing changes
+
+See [`CONTRIBUTING.md`](../CONTRIBUTING.md) for details on how to contribute to
```

Comment:
Not the correct path. Should be `../../../CONTRIBUTING.md`.

Reply:
Fixed.